src/types/any-alias.d.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+type AnyDuringTs42Migration = any;
```

Comment:
Could you add a comment on what this is for, and when (what would we need to check) we expect we can remove it?

If this is just to illustrate and #7078 is the PR you plan to land, adding the comment there is fine too/instead.

Reply:
Thanks for the suggestion. How does this look?